### PR TITLE
Changed/fixed a few things

### PR DESCRIPTION
**Pull request by LukasmanMHdude**
Changed some spelling/grammatical errors, as well as added the last "don't" and "join our discord". (In readme.md)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,27 +1,29 @@
...
+This is the Minehut Meta GitHub repository. Here, you can tell us about your features and ideas that you want to be added to Minehut and report bugs that need to be squashed. We use GitHub's Issue Tracker feature to track bug reports and feature requests (suggestions).
 
 We understand you might not have much experience using GitHub, and that's fine! We'll do our best to help you navigate this tracker. We can start by introducing some key terms you'll be hearing a lot:
 - **Issue**: A feature request or **bug** report.
 - **Repository** or **repo**: Refers to the group of Minehut Meta pages. Includes the **issue** tracker and documents (like this one).
-- **Bug**: Any piece of unintended behaviour anywhere on Minehut. Basically, if something isn't working how it should, it's a bug.
+- **Bug**: Any piece of unintended behavior anywhere on Minehut. Basically, if something isn't working how it should, it's a bug.
```
Comment (PundaChen) [inline README.md:19]: What's up with this one? Behaviour is correct so I'm not sure why you changed it.

File: README.md [hunk 2]
```diff
@@ -1,27 +1,29 @@
...
 - **Valid**: If an **issue** is correct. Reproducible? Correct information? Actually an issue?
 - **Reproducible**: If a **bug** report can be copied or induced by other people. A bug report must be **reproducible** to be **valid**.
 
 Now that you're an expert on the technical jargon we use to manage issues, you're one step closer to making your first issue!
 
 ## Community Expectations
-We expect members of the community participating in conversations here to act in a civil manner and to keep this forum productive and enjoyable. Users who break rules will have their messages deleted and they may be permanently banned from participating depending on the circumstances.
+We expect members of the community participating in conversations here to act civilly and to keep this forum productive and enjoyable. Users who break rules will have their messages deleted and they may be permanently banned from participating depending on the circumstances.
```
Comment (PundaChen) [inline README.md:26]: Personally think that in a civil manner sounds better.

File: README.md [hunk 3]
```diff
@@ -30,7 +32,7 @@ We expect members of the community participating in conversations here to act in
 - Give constructive feedback to ideas
 - Search the issue tracker to ensure that your issue has not already been submitted before making a new post (use the search bar on the [Issues page](https://github.com/Minehut/Meta/issues))
 - Verify the validity of your issue
-- Be clear, concise and specific in your messages
+- Be clear, concise, and specific in your messages
```
Comment (PundaChen) [inline README.md:35]: This is following the US rules, considering UK rules, the previous message was correct.

File: README.md [hunk 4]
```diff
@@ -45,9 +47,10 @@ We expect members of the community participating in conversations here to act in
...
 
-- Once you're ready to submit your bug report or feature request, head over to [this page](https://github.com/Minehut/Meta/issues/new/choose) to begin the process
-- You can also comment on other issues and participate in other conversations. Remember, if you want to show support or general disagreement of an issue, use 👍 and 👎 reactions. You can also [sort issues by 👍 reactions](https://github.com/Minehut/Meta/issues?q=is%3Aissue+sort%3Areactions-%2B1-desc) (also [by 👎 if you'd like](https://github.com/Minehut/Meta/issues?q=is%3Aissue+sort%3Areactions--1-desc))
+- Once you're ready to submit your bug report or feature request, head over to [this page](https://github.com/Minehut/Meta/issues/new/choose) to begin the process.
+- You can also comment on other issues and participate in other conversations. Remember, if you want to show support or general disagreement of an issue, use 👍 , and 👎 reactions. You can also [sort issues by 👍 reactions](https://github.com/Minehut/Meta/issues?q=is%3Aissue+sort%3Areactions-%2B1-desc) (also [by 👎 if you'd like](https://github.com/Minehut/Meta/issues?q=is%3Aissue+sort%3Areactions--1-desc))
```
Comment (PundaChen) [inline README.md:56]: Following US rules again, there's really no need to change this.

File: README.md [hunk 5]
```diff
@@ -1,27 +1,29 @@
...
 
-We would like to improve communication with our users by providing official channels to report problems and tell us your ideas. This space will be taken as an (experimental) approach to see if you feel comfortable reporting and tracking your issues by means of this tool. This document (called the README) will serve as a starting point for making your first issue.
+We would like to improve communication with our users by providing official channels to report problems and tell us your ideas. This space will be taken as an (experimental) approach to see if you feel comfortable reporting and tracking your issues utilizing this tool. This document (called the README) will serve as a starting point for making your first issue.
+
+A lot of conversations and announcements happen on our discord, which you can join [here](https://discord.gg/pgXMSFh). Many longer conversations about issues happen there, as well as diving into other topics such as changing the staff training process.
```
Comment (MiginsMC) [inline README.md:6]: You sorta repeat yourself here. Also the example you made is very specific

File: README.md (hunk 3 above)
Comment (MiginsMC) [reply README.md:35]: I don't think that follows US/UK rules. Adding a comma before `and` is an optional thing to do in writing, it's especially used when listing lengthy items that may contain the term `and` (I think, correct me if I'm wrong). I don't really mind this change, tbh I think it looks nicer.

File: README.md (hunk 4 above)
Comment (MiginsMC) [reply README.md:56]: As I said above, I'm not contested to this change.

File: README.md (hunk 3 above)
Comment (MiginsMC) [reply README.md:35]: I don't think that follows US/UK rules. Adding a comma before `and` is an optional thing to do in writing, it's especially used when listing lengthy items that may contain the term `and` (I think, correct me if I'm wrong). I don't really mind this change, tbh I think it looks nicer.

File: README.md (hunk 4 above)
Comment (MiginsMC) [reply README.md:56]: As I said above, I'm not contested to this change

**Comment by jellz** [comment]
This might come off a bit harsh, but personally, I think this is just an attempt to gain the "Contributor" flair on GH.

Most of these changes are really low-effort and not worth using, and there is no reason to enforce American English in this file (it was completely fine before).

About the added bullet point about tracking bot issues on the bot repo, I think bot feature requests and bug reports should continue to be submitted here because:
- The other repo does not have the same issue formats and is not equipped to handle the submissions
- The other repo does not get as much attention and therefore will not have as much discussion and voting on feature requests and bug reports

The only change worth considering is the addition of the Discord invite to the README, and personally, I don't think the README should include the Discord invite, as this undermines the GitHub Issue discussions. The Meta Discord's main purpose is for holding testing sessions and discussing other larger changes, not standard feature requests/bug reports. And, even if the Discord invite was to be added to the README, the text should be improved.

**Comment by LukasmanMHdude** [comment]
Yea, I can see where you all are coming from

**Comment by LukasmanMHdude** [comment]
Personally, I made this really because I wanted to add the discord link, then changed a few things when I thought they should be changed. And I also Americanized it, which is debatable if that's really nessesary

**Pull request closed without merging**